tests/modules/kaiju/kaiju2table/main.nf

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { UNTAR             } from '../../../../modules/untar/main.nf'
+include { KAIJU_KAIJU       } from '../../../../modules/kaiju/kaiju/main.nf'
+include { KAIJU_KAIJU2TABLE } from '../../../../modules/kaiju/kaiju2table/main.nf'
+
+workflow test_kaiju_kaiju_single_end {
+
+    input = [
+        [ id:'test', single_end:true ], // meta map
+        file(params.test_data['sarscov2']['illumina']['test_1_fastq_gz'], checkIfExists: true)
+    ]
+    db    = [ [], file(params.test_data['sarscov2']['genome']['kaiju_tar_gz'], checkIfExists: true) ]
+    taxon_rank = "species"
+
+    ch_db = UNTAR ( db )
+    KAIJU_KAIJU ( input, ch_db.untar.map{ it[1] } )
+    KAIJU_KAIJU2TABLE ( KAIJU_KAIJU.out.results, ch_db.untar.map{ it[1] }, taxon_rank )
+}
+
+workflow test_kaiju_kaiju_paired_end {
+
+    input = [
+        [ id:'test', single_end:false ], // meta map
+        [ file(params.test_data['sarscov2']['illumina']['test_1_fastq_gz'], checkIfExists: true),
+          file(params.test_data['sarscov2']['illumina']['test_2_fastq_gz'], checkIfExists: true) ]
+    ]
+    db    = [ [], file(params.test_data['sarscov2']['genome']['kaiju_tar_gz'], checkIfExists: true) ]
+    taxon_rank = "species"
+
+    ch_db = UNTAR ( db )
+    KAIJU_KAIJU ( input, ch_db.untar.map{ it[1] } )
+    KAIJU_KAIJU2TABLE ( KAIJU_KAIJU.out.results, ch_db.untar.map{ it[1] }, taxon_rank )
+}
```

Comment:
```suggestion
```

i don't think this is necessary in this case, the input to KAIJU2TABLE is the same in both tests and you don't specify any other extra parameters